gnocchi/rest/__init__.py

```diff
@@ -665,6 +667,8 @@ def post(self):
         except indexer.NoSuchResource as e:
             abort(404, e)
 
+        return r.metrics
```

Comment:
This issue is driving me nuts. Locally, all the gabbi tests pass, assuming that the metric being created in the test will be the first one to be returned in the list.

When testing using the Gnocchi API, metrics are not returned in alphabetical order, but in the order of creation (last created, first in the list). But I don't get why it is not the case in the gate.